docs/docs/guides/integrate/zitadel-apis/access-zitadel-apis.md

```diff
@@ -0,0 +1,237 @@
+  "token_type": "Bearer",
+  "expires_in": 43199
+}
+```
+
+Use the access_token in the Authorization header to make requests to the ZITADEL APIs.
+In the following example we read the organization of the service user.
+
+```bash
+curl --request GET \
+  --url $CUSTOM-DOMAIN/management/v1/orgs/me \
+  --header 'Authorization: Bearer ${TOKEN}' 
+```
+
+### Client credentials
+
+Get the client id and client secret by
+
+1. navigating to the service user, then
+2. Open **Actions** in the top right corner and click on **Generate Client Secret**
+3. Copy the **ClientID** and **ClientSecret** from the dialog
+
+![Create new service user](/img/console_serviceusers_secret.gif)
+
+With the ClientId and ClientSecret from the prior step, you will need to craft a POST request to ZITADEL's token endpoint.
+
+#### Audience scope
+
+**To access the ZITADEL APIs you need the ZITADEL Project ID in the audience of your token.**
+This is possible by sending a [reserved scope](/apis/openidoauth/scopes) for the audience.
+Use the scope `urn:zitadel:iam:org:project:id:zitadel:aud` to include the ZITADEL project id in your audience
+
+In this step we will authenticate a service user and receive an access_token to use against the ZITADEL API.
+
+#### Basic authentication
+
+When using `client_secret_basic` on token or introspection endpoints, provide an`Authorization` header with a Basic auth value in the following form:
+
+```markdown
+Authorization: "Basic " + base64( formUrlEncode(client_id) + ":" + formUrlEncode(client_secret) )
+```
+
+For an example see the [client secret basic authentication method reference](/docs/apis/openidoauth/authn-methods#client-secret-basic).
+We recommend using an OpenID / OAuth library that handles the encoding for you.
+
+#### Post request
+
+You will need to craft a POST request to ZITADEL's token endpoint:
+
+```bash {6}
+curl --request POST \
+  --url https://$CUSTOM-DOMAIN/oauth/v2/token \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --header 'Authorization: Basic ${BASIC_AUTH}' \
+  --data grant_type=client_credentials \
+  --data scope='openid profile urn:zitadel:iam:org:project:id:zitadel:aud'
+```
+
+where
+
+* `grant_type` should be set to `client_credentials`
+* `scope` should contain any [Scopes](/apis/openidoauth/scopes) you want to include, but must include `openid`. For this example, please include `profile`
+  and `urn:zitadel:iam:org:project:id:zitadel:aud`. The latter provides access to the ZITADEL API.
+
+You should receive a successful response with `access_token`,  `token_type` and time to expiry in seconds as `expires_in`.
+
+```bash
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "access_token": "MtjHodGy4zxKylDOhg6kW90WeEQs2q...",
+  "token_type": "Bearer",
+  "expires_in": 43199
+}
+```
+
+Because the received Token includes the `urn:zitadel:iam:org:project:id:zitadel:aud` scope, we can send it in your requests to the ZITADEL API as Authorization Header.
```

Comment:
```suggestion
Because the received token includes the `urn:zitadel:iam:org:project:id:zitadel:aud` scope, we can send it in your requests to the ZITADEL API as the Authorization Header.
```